libs/novui/src/components/select/Select.types.ts

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Let's move the types into the Select.tsx file. There is no benefit from having them in a separate file.